OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -177,7 +181,8 @@ public MovementType CurrentMovementTypes
 		INotifyMoving[] notifyMoving;
 		INotifyFinishedMoving[] notifyFinishedMoving;
 		IWrapMove[] moveWrappers;
-		bool requireForceMove;
+		public bool RequireForceMove;
+		public bool Immovable;
```

Comment:
Minor wording nit: @abcdefg30 has often bugged me in my PRs that we should add an 'Is' in front of such booleans (like `IsBlocking` below). At the very least for consistency with most other booleans in our code.

Reply:
Fixed.